core/tests/wdio.conf.js

```diff
@@ -329,6 +329,7 @@ exports.config = {
         '-i', process.env.DISPLAY,
         '-g', '300',
         '-loglevel', '16',
+        '-preset', 'ultrafast',
```

Comment:
Maybe leave a comment to explain why this is added.

Reply:
Done!